source/initial_temperature/anelasticity_temperature.cc

```diff
@@ -0,0 +1,421 @@
+                           Patterns::Double(),
+                           "The value of the pressure derivative of shear modulus $\\dmu_U/dP$.");
+        prm.declare_entry ("Viscosity prefactor", "6.22e21",
+        				   Patterns::Double(),
+                           "The value of the viscosity prefactor $\\eta_r$. Units: $Pa s$.");
+        prm.declare_entry ("Activation energy", "462.5e3",
+        				   Patterns::Double(),
+                           "The value of the activation energy $\\E_a$. Units: $J/mol$.");
+        prm.declare_entry ("Activation volume", "7.913e-6",
+        				   Patterns::Double(),
+                           "The value of the activation volume $\\V_a$. Units: $m^3/mol$.");
+        prm.declare_entry ("Solidus gradient", "1.018",
+                           Patterns::Double(),
+                           "The value of the solidus gradient $\\ $dT_s/dz}$. Units: $K/km$.");
+
+
+        Utilities::AsciiDataBase<dim>::declare_parameters(prm,
+                                                          "$ASPECT_SOURCE_DIR/data/initial-temperature/ascii-data/test/",
+                                                          "box_2d_Vs_YT16.txt");
+      }
+      prm.leave_subsection();
+    }
+
+
+    template <int dim>
+    void
+    AnelasticVs2T<dim>::parse_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection ("Initial temperature model");
+      {
+        no_perturbation_depth   = prm.get_double ("Remove temperature heterogeneity down to specified depth");
+        use_yamauchi_takei = prm.get_bool ("Use Yamauchi and Takei parameterization");
+        use_original_model = prm.get_bool ("Use original density and frequency model of Yamauchi and Takei");
+        upper_temperature = prm.get_double("Upper temperature");
+        mu0 = prm.get_double ("Reference shear modulus");
+        dmudT = prm.get_double ("Temperature derivative of shear modulus");
+        dmudP = prm.get_double ("Pressure derivative of shear modulus");
+        viscosity_prefactor = prm.get_double ("Viscosity prefactor");
+        activation_energy = prm.get_double ("Activation energy");
+        activation_volume = prm.get_double ("Activation volume");
+        solidus_gradient = prm.get_double ("Solidus gradient");
+
+        Utilities::AsciiDataBase<dim>::parse_parameters(prm);
+      }
+      prm.leave_subsection();
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace InitialTemperature
+  {
+    ASPECT_REGISTER_INITIAL_TEMPERATURE_MODEL(AnelasticVs2T,
+                                              "anelastic Vs to temperature",
+                                              "Implementation of a model in which the initial temperature is calculated "
+                                              "from files containing absolute shear wave velocity (Vs) data in ascii format. "
+                                              "This plug-in allows you to select from a number of different models that"
```

Comment:
If there are several options, mention them here shortly (e.g. their publications), if there is currently just one, them just say it currently only implements one conversion.

Reply:
I've now clarified that this plugin only implements the Yamauchi & Takei 2016 version.